tests/layer_tests/tensorflow_tests/test_tf_MatrixInverse.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestMatrixInverse(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'input:0' in inputs_info
+        input_shape = inputs_info['input:0']
+        inputs_data = {}
+        inputs_data['input:0'] = self._generate_invertible_matrix(input_shape)
+        return inputs_data
+    
+    def _generate_invertible_matrix(self, shape):
+        while True:
+            matrix = np.random.rand(*shape).astype(np.float32)
+             # if the matrix has correct rank, and the determinat is not zero, then it is invertible
+            if np.linalg.matrix_rank(matrix) == shape[0] and np.linalg.det(matrix) != 0:
+                return matrix
+
+    def create_matrix_inverse_net(self, input_shape):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            input_tensor = tf.compat.v1.placeholder(np.float32, input_shape, 'input')
+            output = tf.raw_ops.MatrixInverse(input=input_tensor, adjoint=False)  # Adjust 'adjoint' as needed
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    test_data_basic = [
+        dict(input_shape=[2, 2]),
+        dict(input_shape=[3, 3]),
+        dict(input_shape=[4, 4]),
```

Comment:
please add cases when `input_shape=[2, 4, 4]` with several batches of square matrices which will be inverted separately

Reply:
Thx, added